### ISSUE TITLE
ardop: Non-ARQ data interpreted as received data

### ISSUE THREAD

**Issue by martinhpedersen** [la5nta/pat#139]
I've received several reports of `[ConReq500M: KB3OMM > AB3XYZ]` strings getting passed through as data coming from the remote connected station. The data is often interpreted as an invalid B2F SID line, and causes Pat to abort the session.

It turns out that both ARDOP_win and ARDOPc is sending these strings as ARQ frames when the TNC is in idle state. This is IMHO a violation of the host interface specification they have provided:
> The “d:” identifies this as a data frame from the TNC. ”ARQ” indicates this is data received from a connected (ARQ) session and should be error free. ”FEC” indicates this is data received from an unconnected session ( a monitored FEC or monitored ARQ frame) and should be error free. 

I've been informed by John (ARDOPc author) that the current behavior is as intended, but I have not been able to get a confirmation from Rick (ARDOP_win author and API designer) that they are in fact planning to update the interface specification. As I understand it, Rick (ARDOP_win author) has the final say in the API specifications.

To summarize; There are discrepancies between the ARDOP interface specifications and the actual implementations, and I am unfortunately unable to get confirmations from both authors on how to proceed.

I don't see any other solution than to update the code as per John's recommendation and hope that the specification document is updated soon.
